src/utils/AstUtils.ts

```diff
@@ -110,44 +113,24 @@ export module AstUtils {
         /* tslint:enable:no-console */
     }
 
-    export function isPrivate(node: ts.Node): boolean {
-        /* tslint:disable:no-bitwise */
-        if ((<any>ts).NodeFlags.Private !== undefined) {
```

Comment:
The modifier flag types were moved out of the `NodeFlags` enum just before TypeScript 2.1. Since the minimum version of TypeScript specified in the peer dependencies here is 2.1.0, I figured it was time to remove this and just look at the `ModifierFlags`.